tests/mobile/test_article.py

```diff
@@ -5,13 +5,13 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from pages.mobile.page_provider import PageProvider
+from pages.mobile.page_provider import MobilePageProvider
```

Comment:
You can undo this and all of the other changes below which is just changing the class name to `MobilePageProvider`, if you follow my suggestion in https://github.com/mozilla/sumo-tests/pull/320/files#r44674984